GitUI/UserControls/BlameControl.cs

```diff
@@ -352,10 +397,11 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
                 lastCommit = line.Commit;
             }
 
-            return (gutter.ToString(), body.ToString());
+            return (gutter.ToString(), body.ToString(), gitBlameDisplays);
```

Comment:
With the above comment, we have to do this:
```suggestion
            return (gutter.ToString(), body.ToString(), gitBlameDisplays ?? Array.Empty<GitBlameEntry>());
```

Reply:
not needed because that won't be null here.